src/test/java/no/nav/tag/dittNavArbeidsgiver/services/altinn/AltinnServiceTest.java

```diff
@@ -0,0 +1,24 @@
+package no.nav.tag.dittNavArbeidsgiver.services.altinn;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("dev")
+public class AltinnServiceTest {
+
+    @Autowired
+    private AltinnService altinnService;
+
+    @Test(expected = AltinnException.class)
```

Comment:
Kanskje legge til en test som sjekker det positive tilfellet også? Slik det er nå kan `hentOrganisasjoner` bare alltid kaste AltinnException, og alle tester kjører grønt.

Reply:
fikk bruk for det nå. har lagt til en test for dette :-)